lib/hls/hls_parser.js

```diff
@@ -3616,6 +3624,130 @@ shaka.hls.HlsParser = class {
     return [startByte, endByte];
   }
 
+  /**
+   * @param {!Array.<!shaka.hls.Tag>} tags
+   * @private
+   */
+  processDateRangeTags_(tags) {
+    const initialProgramDateTime =
+        this.presentationTimeline_.getInitialProgramDateTime();
+    if (!initialProgramDateTime ||
+        this.config_.hls.ignoreManifestProgramDateTime) {
```

Comment:
You should update the description of `ignoreManifestProgramDateTime` in the externs, to mention that it also causes us to ignore `EXT-X-DATERANGE` tags.

Reply:
Done!